src/components/menu/position/position.ts

```diff
@@ -0,0 +1,214 @@
+// (C) 2007-2018 GoodData Corporation
```

Comment:
I would suggest renaming the directory of this file to "positioning" and the file to "positioningCalculations.ts"

Reply:
Renamed